libs/shared/ui-components/src/Hero/types.ts

```diff
@@ -29,8 +29,8 @@ export type THeroProps = {
   variant: HeroVariant;
   title?: string;
   subTitle?: string;
-  imageSrc: string;
-  imageAlt: string;
+  imageSrc?: string;
+  imageAlt?: string;
```

Comment:
As imageSrc and imageAlt are now optional, you might need to add condition to Picture in Hero to avoid ts error